.github/workflows/docker.yml

```diff
@@ -15,6 +15,10 @@ on:
     # This needs to be declared explicitly so that the job is actually
     # run when moved out of draft.
     types: [opened, synchronize, reopened, ready_for_review]
+  pull_request_target:
```

Comment:
Does this one not need branches?

Reply:
For consistency it should, I'll address it in another PR given it's outside of the scope of this one, and I'd like to have an easy way of reverting this if needed.